nixos/platform/acme.nix

```diff
@@ -1,22 +1,44 @@
 { config, pkgs, lib, ... }:
 let
   inherit (config) fclib;
+  checkCert = pkgs.writeShellScript "check-local-ssh-cert" ''
+    certificate_path="$1"
+    critical_sec=$((14 * 24 * 3600))
+    warning_sec=$((25 * 24 * 3600))
```

Comment:
These **could** be made configurable, but as we're transitioning away from the previous check, that one also had hard-coded intervals.